package.json

```diff
@@ -61,13 +61,16 @@
     "export_tests_to_site": "cp test/unit/*.js ../fabricjs.com/test/unit && cp -r test/visual/* ../fabricjs.com/test/visual && cp -r test/fixtures/* ../fabricjs.com/test/fixtures && cp -r test/lib/* ../fabricjs.com/test/lib",
     "all": "npm run build && npm run test && npm run test:visual && npm run lint && npm run lint_tests && npm run export_dist_to_site && npm run export_tests_to_site",
     "testem": "testem .",
-    "testem:ci": "testem ci"
+    "testem:ci": "testem ci",
+    "react_sandbox:start": "node test/react_sandbox start",
+    "react_sandbox:deploy": "node test/react_sandbox deploy"
   },
   "optionalDependencies": {
     "canvas": "^2.6.1",
     "jsdom": "^15.2.1"
   },
   "devDependencies": {
+    "axios": "^0.21.1",
```

Comment:
can we move axios in the template package json? or start with a template that do no require axios?

Reply:
DONE